src/applications/mhv-medical-records/tests/components/LabsAndTestsListItem.unit.spec.jsx

```diff
@@ -58,7 +58,7 @@ describe('LabsAndTestsListItem component', () => {
     const recordDetailsLink = screen.getByText(
       'POTASSIUM:SCNC:PT:SER/PLAS:QN:, SODIUM:SCNC:PT:SER/PLAS:QN:',
       {
-        selector: 'a',
+        selector: 'span',
```

Comment:
Remove the change from mhv-medical-records app, as we are going to delete it.

Reply:
I changed it back to `a`.